easybuild/easyconfigs/b/beagle-lib/beagle-lib-2.1.2-goolf-1.7.20.eb

```diff
@@ -2,6 +2,7 @@ easyblock = 'ConfigureMake'
 
 name = 'beagle-lib'
 version = '2.1.2'
+versionsuffix = '-Java-%(javaver)s'
```

Comment:
@boegel beast2 requires Java8. See what happens if I try to execute it with java7

```
[escobar@login18 easybuild-easyconfigs]$ beast --help
BEAST requires Java version 8,
but the current version is 7.
```